src/backend/distributed/planner/multi_colocated_subquery_planner.c

```diff
@@ -0,0 +1,783 @@
+/*
+ * multi_colocated_subquery_planner.c
```

Comment:
For the time being, we changed to `multi_planner_utils.c`. Does that work for you?